Makefile

```diff
@@ -3,11 +3,19 @@ OUTPUT_DIR :=_output
 # Constants
 GOPATH := $(shell go env GOPATH)
 
+# Version
+revision:=$(shell git rev-parse --short HEAD)
+build_time:=$(shell date +%D@%T)
+version:=$(revision)-$(build_time)
```

Comment:
I like the idea of having the version set by the git revision and build time, but I don't want to lose the semantic version info (0.1.0). In my opinion, that's very user friendly and can allows for quick inspection/verification.

I would suggest having it as something like: 

version:=$(semantic-version)-$(revision)-$(build_time)

What do you think?

Reply:
added